pkgs/by-name/st/stalwart-mail/package.nix

```diff
@@ -26,6 +27,10 @@ let
   # https://github.com/stalwartlabs/mail-server/issues/407
   rocksdb = rocksdb_8_11;
   version = "0.8.5";
+  webadmin = fetchurl {
+    url = "https://github.com/stalwartlabs/webadmin/releases/download/v0.1.8/webadmin.zip";
```

Comment:
Not sure how that works out in combination with the update script. Was there a reason that you opted against a separate package with its own file, attr and update script?

I have something similar like that I may personal nix configuration but opted against upstreaming it, as i was not (yet) sure if just providing the binary release is intended.

Not sure how others opinion is about that, I personally would drop that into a sepat package.
In my config I also opted in providing it as not $out output as it should not go into systemPackages.

Reply:
Building it from source seems a bit tricky (involves cargo, node package, tailwind etc) but I could move it into webadmin.nix beside the package.nix of stalwart?

Reply:
like i said i am not 100% sure here. lets check if some other stalwart user has an opinion how to approach here.

Reply:
The steps seem pretty straight forward https://github.com/stalwartlabs/webadmin/blob/main/.github/workflows/build.yml#L26-L46

Also the release cycle seems to be not tied to the actual server.